apps/api/src/app/environments-v1/usecases/output-renderers/hydrate-email-schema.usecase.ts

```diff
@@ -95,92 +82,6 @@ export class HydrateEmailSchemaUseCase {
     return !!(node.type === 'variable' && node.attrs && 'id' in node.attrs && typeof node.attrs.id === 'string');
   }
-
-  private getResolvedValueForPlaceholder(
-    masterPayload: PreviewPayload,
-    node: TipTapNode & {
-      attrs: { each: string };
-    },
-    itemPointerToDefaultRecord: Record<string, string>,
-    placeholderAggregation: PlaceholderAggregation
-  ) {
-    let resolvedValueIfFound = this.getValueByPath(masterPayload, node.attrs.each);
-
-    if (!resolvedValueIfFound) {
-      resolvedValueIfFound = [
-        this.buildElement(itemPointerToDefaultRecord, '1'),
-        this.buildElement(itemPointerToDefaultRecord, '2'),
-      ];
-    }
-    placeholderAggregation.nestedForPlaceholders[`{{${node.attrs.each}}}`] =
-      this.buildNestedVariableRecord(itemPointerToDefaultRecord);
-
-    return resolvedValueIfFound;
-  }
-
-  private buildNestedVariableRecord(itemPointerToDefaultRecord: Record<string, string>) {
-    const transformedObj: Record<string, string> = {};
-
-    Object.entries(itemPointerToDefaultRecord).forEach(([key, value]) => {
-      transformedObj[value] = value;
-    });
-
-    return transformedObj;
-  }
-  private collectAllItemPlaceholders(nodeExt: TipTapNode) {
-    const payloadValues = {};
-    const traverse = (node: TipTapNode) => {
-      if (node.type === 'for') {
-        return;
-      }
-      if (this.isPayloadValue(node)) {
-        const { id } = node.attrs;
-        payloadValues[`${node.attrs.id}`] = node.attrs.fallback || `{{item.${id}}}`;
-      }
-      if (node.content && Array.isArray(node.content)) {
-        node.content.forEach(traverse);
-      }
-    };
-    nodeExt.content?.forEach(traverse);
-
-    return payloadValues;
-  }
-
-  private getValueByPath(masterPayload: Record<string, any>, placeholderRef: string): any {
-    const keys = placeholderRef.split('.');
-
-    return keys.reduce((currentObj, key) => {
-      if (currentObj && typeof currentObj === 'object' && key in currentObj) {
-        return currentObj[key];
-      }
-
-      return undefined;
-    }, masterPayload);
-  }
-
-  private buildElement(itemPointerToDefaultRecord: Record<string, string>, suffix: string) {
-    const mockPayload: Record<string, any> = {};
-    Object.keys(itemPointerToDefaultRecord).forEach((key) => {
-      const keys = key.split('.');
-      let current = mockPayload;
-      keys.forEach((innerKey, index) => {
-        if (!current[innerKey]) {
-          current[innerKey] = {};
-        }
-        if (index === keys.length - 1) {
-          current[innerKey] = itemPointerToDefaultRecord[key] + suffix;
-        } else {
-          current = current[innerKey];
-        }
-      });
-    });
-
-    return mockPayload;
-  }
-
-  private isPayloadValue(node: TipTapNode): node is { type: 'payloadValue'; attrs: { id: string; fallback?: string } } {
-    return !!(node.type === 'payloadValue' && node.attrs && typeof node.attrs.id === 'string');
-  }
```

Comment:
not needed since we are removing related stuff here: https://github.com/novuhq/novu/pull/7484